Source/santa/BUILD

```diff
@@ -70,7 +70,7 @@ macos_application(
     minimum_os_version = "10.15",
     provisioning_profile = select({
         "//:ci_build": None,
-        "//conditions:default": "Santa_Dev.provisionprofile",
+        "//conditions:default": "//profiles:santa_dev",
```

Comment:
Is there any way we could move this select() into the profiles package?

Reply:
Tried this and it's not great since you can't have the select apply as the result of the label. The issue here is that we need either `None` for the rule or a file. Filegroup can't return an empty list and satisfy the `provisioning_profile`.

Reply:
Ack, was worth asking.